src/main/java/com/booster/server/VocabularyEntryRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.booster.server;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
You can omit @Repository annotation as you extend JpaRepository. Spring will create this repository anyway, but it depends on code style and preferences.

Reply:
Agree, less but better